assets/informations/rapson.json

```diff
@@ -0,0 +1,9 @@
+{
+    "building": "Rapson Hall, Ralph",
```

Comment:
Not sure who Ralph is

Reply:
When I was searching for all the information, on the UMN site, the building was named "Rapson Hall, Ralph", I can change it to "Rapson Hall" if you would like.

Reply:
I see that same page. The hall is named after Ralph Rapson. However, we have been using a last name basis for other buildings (Keller, Coffman, etc.). Please rename to Rapson Hall.